shell/devconfig.sh

```diff
@@ -96,6 +96,20 @@ DEVENV_DEPLOY_ENVIRONMENT=local_development "$DIR/build-jsonnet.sh" show | yq -r
     rm "$tmpFile" "$mergedFile" >/dev/null 2>&1 || true
   done
 
+info "Fetching non-Vault Secret(s)"
+# Why: `$secretName` is intended as a yq variable not a shell variable.
+# shellcheck disable=SC2016
+DEPLOY_TO_DEV_ENVIRONMENT=local_development "$DIR/build-jsonnet.sh" show | yq -r 'select(.kind == "Secret") | .metadata.name as $secretName | (.data | to_entries[] | [$secretName, .key, .value] | @tsv)' | "$envsubst" |
```

Comment:
Another jq expert ❤️

Reply:
Not really.  This one-liner took me 40 minutes and I was starting from an example that already had most of what I needed.  :/